### ISSUE TITLE
feat: Make sfpowerscripts compatible with the newest sfdx Apple Silicon version

### ISSUE THREAD

**Issue by petter-eikeland** [dxatscale/sfpowerscripts#1248]
**Describe the feature**
Be able to install the sfpowerscripts plugin on the newest Apple Silicon sfdx version

**To Reproduce**
Uninstall current sfdx CLI
Download newest sfdx Apple Silicon version on https://developer.salesforce.com/tools/sfdxcli
Install sfpowerscripts plugin using `sfdx plugins:install @dxatscale/sfpowerscripts`

**Expected behavior**
sfpowerscripts plugin installed successfully

**Actual behavior**
Install plugin failed (see screenshot below)

**Screenshots**
![image](https://user-images.githubusercontent.com/97173615/220915582-4f40d650-640e-40fa-b182-9aa861d8e2b4.png)

**Platform Details (please complete the following information):**
 - OS: Mac OS 13.2.1
 - Version [e.g. CLI Version  eg: 1.6.6]  N/A
 - Salesforce CLI(sfdx cli) Version:  7.188.1
 - CI Platform: N/A

**Comment by petter-eikeland** [comment]
I didn't try downloading sfpowerscripts though and manually linking it. Perhaps that would work?

**Comment by azlam-abdulsalam** [comment]
@petter-eikeland We dont have a M2 machine handy..  Can you try export DEBUG=* and try running any of the sfpowerscripts command, and see which import is breaking

**Comment by ethan-sargent** [comment]
@petter-eikeland I've had a similar issue before with an Intel Mac (2019). 
I had to do a full uninstallation and reinstallation of yarn for the installation to work correctly.

Cloning the repo and linking it may work - it may also give a clearer error message.

Could you confirm your node, npm and yarn versions, and whether you're using a package manager to manage installations and versioning of JS tooling like [Volta](https://volta.sh/) or [nvm](https://github.com/nvm-sh/nvm)?

**Comment by petter-eikeland** [comment]
@azlam-abdulsalam Yeah, it's an M1 Max MBP so not an M2 specific issue. I tried again now, and now the plugin installed without any issue! Not sure why

Just to be clear, the sfpowerscripts & sfdx commands did not suddenly stop working. It was the new Apple Silicon version of sfdx that was released 2 weeks ago that I tried that suddenly throwed the yarn error. 
![Screenshot 2023-03-03 at 09 37 33](https://user-images.githubusercontent.com/97173615/222672226-83e2fc37-6990-4213-bcb3-4aaf8d3827b3.png)
